concordium-contracts-common/src/types.rs

```diff
@@ -392,6 +393,43 @@ impl AccountBalance {
     pub fn available(&self) -> Amount { self.total - cmp::max(self.locked, self.staked) }
 }
 
+#[derive(Debug)]
+pub enum AccountKind {}
+
+/// The minimum number of credentials that need to sign any transaction coming
+/// from an associated account.
+pub type AccountThreshold = Threshold<AccountKind>;
+
+#[derive(Debug)]
+pub enum SignatureKind {}
+
+/// The minimum number of signatures on a credential that need to sign any
+/// transaction coming from an associated account.
+pub type SignatureThreshold = Threshold<SignatureKind>;
```

Comment:
I think the name needs to tie it to the credential, just like `AccountThreshold` represents the threshold per account.
```suggestion
pub type CredentialThreshold = Threshold<CredentialKind>;
```

Reply:
I see now that this type is also used for accounts in base, and changing it would be breaking, so nvm